tsconfig.json

```diff
@@ -12,6 +13,12 @@
     "noEmit": true,
     "jsx": "preserve",
     "allowSyntheticDefaultImports": true,
+    "paths": {
+      "@/infrastructure/*": ["infrastructure/*"],
+      "@/presentation/*": ["presentation/*"],
+      "@/domain/*": ["domain/*"],
+      "@/application/*": ["application/*"]
```

Comment:
did you tried this?
```suggestion
      "@/*": ["./src/*"],
```

Reply:
it is not working in my IDE

Reply:
but i can simplify vite.config

Reply:
Try changing the `include` property:

```
"./src/**/*.ts",
"./src/**/*.vue",
```

and vite alias:
```
'@': path.resolve(__dirname, './src'),
```

Reply:
`'@/': '/src/',`, i can make this in vite alias
Changing `include` property does not help